staticaddr/address/interface.go

```diff
@@ -18,7 +18,8 @@ var (
 type Store interface {
 	// CreateStaticAddress inserts a new static address with its parameters
 	// into the store.
-	CreateStaticAddress(ctx context.Context, addrParams *Parameters) error
+	CreateStaticAddress(ctx context.Context, addrParams *Parameters,
+		currentHeight int32) error
```

Comment:
Why a separate argument `currentHeight` is needed? `addrParams` has new field `InitiationHeight`.

Reply:
removed `currentHeight` and use the `InitiationHeight`.